packages/openneuro-app/src/scripts/search/search-input.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useRef } from 'react'
+import { useHistory } from 'react-router-dom'
+
+const SearchInput = () => {
+  const inputRef = useRef(null)
```

Comment:
This is a ref because otherwise autocomplete causes a lot of DOM churn. Autocomplete isn't implemented here but that's why I ended up with a seemingly useless ref.